### PR TITLE
Update Gorush container image

### PR DESCRIPTION
**Pull request by yokotaso**
Update gorush image which is patch build version. 

Patch PR:  https://github.com/appleboy/gorush/pull/568, https://github.com/cybozu/gorush/pull/1

### CODE REVIEW

File: gorush/Dockerfile [hunk 1]
```diff
@@ -3,10 +3,10 @@
 # Stage1: build from source
 FROM quay.io/cybozu/golang:1.15-bionic AS build
```
Comment (ymmt2005) [inline gorush/Dockerfile:4]: The upstream has moved to Go 1.16, right?
https://github.com/appleboy/gorush/commit/ae91ee7be1a292eed86cf2f752c442799a1403f3
```suggestion
FROM quay.io/cybozu/golang:1.16-focal AS build
```

File: gorush/TAG [hunk 2]
```diff
@@ -1 +1 @@
-1.12.0.2
+1.13.0.3
```
Comment (ymmt2005) [inline gorush/TAG:1]: You may reset the image revision to 1 when the other versions change.
```suggestion
1.13.0.1
```

File: gorush/Dockerfile [hunk 3]
```diff
@@ -3,10 +3,10 @@
...
 
-ARG GORUSH_VERSION="1.12.0"
+ARG GORUSH_VERSION="1.13.0.3"
```
Comment (ymmt2005) [inline gorush/Dockerfile:6]: I guess this is your fork and is a tentative release.
Why don't you tag it as `1.13.0.cybozu.1` or likewise?

**Review by ymmt2005 — CHANGES_REQUESTED** [review]

File: gorush/Dockerfile (hunk 1 above)
Comment (yokotaso) [reply gorush/Dockerfile:4]: I merge patch into v1.13.0. Therefore failed to build with gorush 1.16

Comment (ymmt2005) [reply gorush/Dockerfile:4]: I see.

File: gorush/TAG (hunk 2 above)
Comment (yokotaso) [reply gorush/TAG:1]: - v1.13.0.1 is used by https://github.com/cybozu/neco-containers/pull/440
- v1.13.0.2 is used by https://github.com/cybozu/neco-containers/pull/464

Comment (ymmt2005) [reply gorush/TAG:1]: Oh, so why was this line `1.12.0.2`?

Comment (yokotaso) [reply gorush/TAG:1]: After update gorush v1.13.0, I happend to gorush degrade. So I downgrade version gorush v1.12.0

Ref: https://github.com/cybozu/neco-containers/pull/468

After all, I found workaround of gorush degrade. In maneki-apps I use gorush v1.13.0

**Review by ymmt2005 — APPROVED** [review]
LGTM

**Pull request merged**